lib/engine/game/g_18_al.rb

```diff
@@ -18,6 +18,12 @@ class G18AL < Base
 
       def setup
         setup_company_price_50_to_150_percent
+
+        @corporations.each do |corporation|
+          corporation.abilities(:destination) do |ability|
+            ability.value = @hexes.detect { |h| h.name == ability.hex }.location_name
```

Comment:
use find

Reply:
As far as I can tell detect and find are equivalent, and it seems find does call detect internally. But sure, I will switch in the next push.